examples/library-test/service-template/service.yaml

```
@@ -0,0 +1,11 @@
+tosca_definitions_version: tosca_simple_yaml_1_3
+
+imports:
+  - /library/steampunk.valves/main.yaml
+  - /library/steampunk.gears/main.yaml
+  - library/types.yaml
+
+topology_template:
+  node_templates:
+    dummy:
+      type: tosca.nodes.Root
```

I just went through a lot of gears and valves here :smiley: I would propose this `topology_template` to be improved a little bit because now it uses just `dummy` node that derives from tosca root node. You may use gear and valve node TOSCA definitions you declared in other files that are already imported here.

My suggestion would be something like that:

```suggestion
topology_template:
  node_templates:
    dummy:
      type: tosca.nodes.Root
      
    steam_valve:
      type: steampunk.valves.Root
      
    landing_gear:
      type: steampunk.gears.Root

    my_big_fella:
      type: big_valve
```

What do you think?

---

Thank you, @anzoman , sounds good 👍